ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -276,6 +276,11 @@ private SafeFuture<BlockImportResult> handleBlockAndBlobsSidecarImport(
                     failedPayloadExecutionSubscribers.deliver(
                         FailedPayloadExecutionSubscriber::onPayloadExecutionFailed, block);
                     break;
+                  case FAILED_BLOBS_AVAILABILITY_CHECK:
+                    // TODO: we cannot invalidate the block root since it passed the checks
```

Comment:
I think we should reflect something like this here:
TODO: trigger the fetcher in the case the coupled `BeaconBlockAndBlobsSidecar` contains a valid block but the BlobsSidecar validation fails like we do with `pendingBlocks`